src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -34,6 +34,17 @@
 from Utils.Pipeline import Pipeline, Functor
 from WMCore.WMException import WMException
 from WMCore.Services.LogDB.LogDB import LogDB
+from WMCore.Services.WMStatsServer.WMStatsServer import WMStatsServer
+from WMCore.MicroService.Unified.Common import findParent
+
+
+class MSRuleCleanerResolveParentError(WMException):
+    """
+    resolve Parent Exception Class for MSRuleCleaner Module in WMCore MicroServices
```

Comment:
I feel bad saying it, but you won't know unless someone says it...
Can I ask you to use capital letters only where it makes sense? One day we shall have it all automatically documented and available, and we all like reading things nicely written ;)

To be clear, my sentence would be:
```
WMCore exception class for the MSRuleCleaner module, in WMCore MicroServices,
used to signal if an error occurred during parent dataset resolution step.
```

This should be considered anywhere, be it a comment, docstring, log record, etc.

Reply:
Done

Reply:
Done